crates/bitwarden-core/src/client/mod.rs

```diff
@@ -14,11 +14,12 @@ pub mod login_method;
 #[cfg(feature = "secrets")]
 pub(crate) use login_method::ServiceAccountLoginMethod;
 pub(crate) use login_method::{LoginMethod, UserLoginMethod};
-#[cfg(feature = "internal")]
-mod flags;
+#[allow(missing_docs)]
+pub mod flags;
```

Comment:
```suggestion
mod flags;
```

You don't need to make this module pub, as you're already doing `pub use flags::Flags` a few lines after it.

Also both lines should be able to be tagged with the internal flag, the wasm crate is enabling it.
```
#[cfg(feature = "internal")]
mod flags;

#[cfg(feature = "internal")]
pub use flags::Flags;
```

Reply:
Gotcha, thanks! Fixed in [1484c3c](https://github.com/bitwarden/sdk-internal/pull/369/commits/1484c3c25972ba5d6715887188781d1b9cd5fb6d)

Reply:
We intentionally didn't make flags public for mobile in order to avoid removal of flags from being a breaking change.